src/constants/objectHelper.js

```diff
@@ -0,0 +1,99 @@
+angular.module('bicker_router').constant('ObjectHelper', {
```

Comment:
I think this whole file could be replaced with lodash stuff, doesn't have to be done now though...